contrib/fwupd.spec.in

```diff
@@ -195,6 +198,26 @@ update operation. This subpackage is probably not required on embedded hardware
 or server machines.
 %endif
 
+%if 0%{?qubes_packages}
+%package qubes-dom0
+Summary: fwupd wrapper for Qubes OS - dom0 scripts
+Requires:   cabextract
```

Comment:
is this still true? can we use gcab for this instead?

Reply:
I swapped cabextract to gcab.